.semaphore/semaphore.yml

```
@@ -15,11 +15,12 @@ blocks:
         commands:
           - checkout
           - cache restore client-node-modules-$SEMAPHORE_GIT_BRANCH-$(checksum yarn.lock),client-node-modules-$SEMAPHORE_GIT_BRANCH,client-node-modules-master
+          - nvm use 10.15.1
```

```suggestion
          - nvm use
```
this will just refer to the existing `.nvmrc`

---

Nice, thanks!